content/en/docs/instrumentation/js/instrumentation.md

```diff
@@ -262,9 +287,8 @@ It can be a good idea to record exceptions when they happen. It's recommended to
 ```js
 try {
   doWork();
-} catch (eexrr) {
-  span.recordException(ex),
-  span.setStatus({ code: otel.SpanStatusCode.ERROR })
+} catch (ex) {
+  span.recordException(ex), span.setStatus({ code: otel.SpanStatusCode.ERROR });
```

Comment:
I think the statement trailing comma is a kinda typo for semicolons. It is unusual for us to use the comma operator for two distinct expressions.

Reply:
```suggestion
  span.recordException(ex);
  span.setStatus({ code: otel.SpanStatusCode.ERROR });
```